### PR TITLE
Updating blog readme, adding new blog posts and adding hyperlinks to …

### PR DESCRIPTION
**Pull request by rakatyal**
…blog post authors

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -129,27 +129,22 @@ Ask for help on the IRC channel: #cordova on irc.freenode.net.
...
     # Set a marker where the summary on the home page should stop displaying. Add the following html comment line to your md file at the desired cutoff point:
     <!--more-->
 
-    # in the front matter of your blog entry, set the `date:` field to the desired date that you want to appear near the title. Be aware that the date (explicit here or implied via the filename) will be used to generate the relative path to this html file (i.e., "/announcements/2014/09/22/cordova-361.html"), as will the `categories:` front matter value.
+    # In the front matter of your blog entry, set the `date:` field to the desired date that you want to appear near the title. Be aware that the date (explicit here or implied via the filename) will be used to generate the relative path to this html file (i.e., "/announcements/2014/09/22/cordova-361.html"), as will the `categories:` front matter value.
     date: 2014-09-22
     categories: releases
 
-    # git commit your changes to apache-blog-posts
+    # Preview it locally by running the site using gulp
```
Comment (dblotsky) [inline README.md]: Please make this whole block not be a code block, except for the pieces of code. It was just poorly formatted before.

Comment (rakatyal) [inline README.md]: Yup.

File: README.md [hunk 2]
```diff
@@ -129,39 +129,26 @@ Ask for help on the IRC channel: #cordova on irc.freenode.net.
...
-Use the grunt scripts!
-
-    # Clone apache-blog-posts repo if you haven't already
-    $ grunt cloneBlog
-
-    # Create your blog posts in the apache-blog-posts (or on github) directory, and share with other for review via `git commit`. If creating a new md file, use an earlier one as a template.
-
-    # pull down the latest shared edits of apache-blog-posts
+1. Pull down the latest website codebase for the current posts
     git pull
```
Comment (dblotsky) [inline README.md]: Markdown needs an empty line before code, otherwise it bunches it into the list item.

**Comment by dblotsky** [comment]
Merged. Feel free to close and delete.

**Pull request closed without merging**